### PR TITLE
Description of the developed functionalities fixes #94

### LINKED ISSUE #94: Description of the developed functionalities

### PR DESCRIPTION
**Pull request by eanker**
A first version of the functionalities of the system.

Feedback is more than welcome.

### CODE REVIEW

File: Documentation/General/finalReport/Description.tex [hunk 1]
```diff
@@ -1 +1,44 @@
-\chapter{Description of the developed functionalities}
\ No newline at end of file
+\chapter{Description of the developed functionalities}
+
+\section{Introduction}
+In this section we will discuss the main functionalities of our system.
+MoodCat can be split up into three main features.
+These functionalities are the ability to play music, chatting and classifying a song.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/Description.tex]: chatting in the rooms*

File: Documentation/General/finalReport/Description.tex [hunk 2]
```diff
@@ -1 +1,44 @@
...
+They will be described in the sections below.
+But before explaining that, we will give a small overview of the structure of the system.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/Description.tex:7]: Good introduction!

File: Documentation/General/finalReport/Description.tex [hunk 3]
```diff
@@ -1 +1,44 @@
...
+
+%Is this needed?
+\section{Small overview}
+When a user comes at website he selects one or more moods he is in.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/Description.tex]: > When a user ~~comes at~~ enters the website

File: Documentation/General/finalReport/Description.tex [hunk 4]
```diff
@@ -1 +1,44 @@
...
+In this section we will discuss the main functionalities of our system.
+MoodCat can be split up into three main features.
+These functionalities are the ability to play music, chatting and classifying a song.
+They will be described in the sections below.
+But before explaining that, we will give a small overview of the structure of the system.
+
+%Is this needed?
+\section{Small overview}
+When a user comes at website he selects one or more moods he is in. 
+Then a list of rooms fitting that selection of moods will show.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/Description.tex]: > ~~show~~ appear.

File: Documentation/General/finalReport/Description.tex [hunk 5]
```diff
@@ -1 +1,44 @@
...
+MoodCat also has a game where a user gets non-rated songs and gets to classify while listening to a maximum 30 seconds snippets of songs.
+
+
+\section{Music playing}
+MoodCat is developed as a music application and thus the ability to play music is very important.
+Although, our music player is a bit different from a normal music player.
+MoodCat is a streaming service, because of that the music is synchronized with the room the player is in.  
+So you can only mute the sound and then the music will keep playing, only with no sound.
+When the internet connection of the user gets down for a few seconds or the connection is slow, when reconnection the music starts again at the moment where the whole room is.\\
+Thus, the music playing functionality of MoodCat can be best described as a radio station.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/Description.tex]: not best described, but you can think of moodcat as a radio station.

File: Documentation/General/finalReport/Description.tex [hunk 6]
```diff
@@ -1 +1,44 @@
...
+
+\section{Chatting}
+In MoodCat there is the functionality to chat with the other people in the room.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/Description.tex]: Its not a funtionality that is just there, its one of our key points. Therefore this should be stated as:

> In Moodcat you can chat with other people in the room.

File: Documentation/General/finalReport/Description.tex [hunk 7]
```diff
@@ -1 +1,44 @@
...
+In order to sent a message the user needs to be logged in, there is no need for creating an account.
+MoodCat uses SoundCloud to login a user.\\
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/Description.tex]: > Users can login with their SoundCloud account.

File: Documentation/General/finalReport/Description.tex [hunk 8]
```diff
@@ -1 +1,44 @@
...
+
+\section{Introduction}
+In this section we will discuss the main functionalities of our system.
+MoodCat can be split up into three main features.
+These functionalities are the ability to play music, chatting in the rooms and classifying a song.
+They will be described in the sections below.
+But before explaining that, we will give a small overview of the structure of the system.
+
+%Is this needed?
+\section{Small overview}
```
Comment (GijsWeterings) [inline Documentation/General/finalReport/Description.tex]: I wouldn't call this section "small overview", Maybe "Introduction to moodcat?"

File: Documentation/General/finalReport/Description.tex [hunk 9]
```diff
@@ -1 +1,44 @@
...
+When a user enters the website he selects one or more moods he is in.
```
Comment (GijsWeterings) [inline Documentation/General/finalReport/Description.tex]: "he is in" can be omitted

**Comment by jwgmeligmeyling** [comment]
Merging now!

**Pull request merged**